examples/credential_metadata_sd_jwt_vc.json

```diff
@@ -0,0 +1,63 @@
+{
+    "credentials_supported": {
+        "IdentityCredentialExampleInOID4VCI": {
+            "format": "vc+sd-jwt",
+            "scope": "IdentityCredential_SD-JWT-VC",
+            "cryptographic_binding_methods_supported": [
+                "did:example"
+            ],
+            "cryptographic_suites_supported": [
+                "ES256"
+            ],
+            "display": [
+                {
+                    "name": "IdentityCredential",
+                    "locale": "en-US",
+                    "background_color": "#12107c",
+                    "text_color": "#FFFFFF"
+                }
+            ],
+            "credential_definition": {
+                "vct": "IdentityCredential",
+                "claims": {
+                    "given_name": {
+                        "display": [
```

Comment:
Could be worth to consider https://github.com/openid/OpenID4VCI/issues/147 here already and transform the display array into an object with the localizations as keys

Reply:
let's deal with that in a separate PR if agreed.